apps/gp2-server/src/routes/public/output.route.ts

```diff
@@ -87,4 +87,9 @@ const mapOutputToPublicOutput = (
     output.subtype === 'Preprints'
       ? output.publishDate
       : undefined,
+  projects: output.projects,
+  identifier: output.doi || output.rrid || output.accessionNumber,
```

Comment:
Here it says in sheets (https://docs.google.com/spreadsheets/d/1t3CUrauL1gLaOeyEDg28NkSQMiYjiINNZ73it4pitBI/edit?gid=1524054492#gid=1524054492) "with link"

I wonder with we should provide the link like https://doi.org/10.1101/2023.11.28.23299090

<img width="1129" alt="Screenshot 2024-08-02 at 13 06 48" src="https://github.com/user-attachments/assets/30077163-28cd-4bd4-9e58-fe384a0ad52a">

Reply:
Thanks! I updated accordingly